launcher/src/backend/ServiceManager.js

```diff
@@ -907,50 +975,68 @@ export class ServiceManager {
     let ELInstalls = tasks.filter((t) => t.service.category === "execution");
     ELInstalls.forEach((t) => {
       let service = this.getService(t.service.service, t.data);
-      t.service.config.serviceID = service.id;
+      if (!Object.hasOwn(t.service, "config")) {
+        t.service.id = service.id;
+      } else {
```

Comment:
Idk exactly what this is for, but this breaks the regular Custom Installation of clients.

(same for CC, VC and Mev)